script.js

```diff
@@ -0,0 +1,130 @@
+import './style.css';
+
+class ToDoList {
+  constructor() {
+    this.toDo = JSON.parse(localStorage.getItem('toDo')) || [];
+    this.clearBtn = document.querySelector('.clear-btn');
+    this.toDoForm = document.querySelector('.to-do-form');
+    this.toDoDiv = document.querySelector('.to-do-container');
+    this.toDoForm.addEventListener('submit', this.addToDo.bind(this));
+    this.clearBtn.addEventListener('click', this.clearCompleted.bind(this));
+    this.render();
+  }
+
+  saveToLocalStorage() {
+    localStorage.setItem('toDo', JSON.stringify(this.toDo));
+  }
+
+  render() {
+    this.toDoDiv.innerHTML = '';
+    this.toDo.forEach((todo) => {
+      const ToDoElement = this.createToDo(todo);
+      this.toDoDiv.appendChild(ToDoElement);
+    });
```

Comment:
Please Use ES6 Modules instead of One Single Class in index.js; It will Help you and Other Developers understanding the code.
Your Script and Functions are not working properly.
I mentioned it in the Comments.